Directory.Build.props

```diff
@@ -1,73 +1,36 @@
-        <Company>DSharpPlus developers</Company>
-        <PackageProjectUrl>https://github.com/DSharpPlus/DSharpPlus</PackageProjectUrl>
-        <RepositoryUrl>https://github.com/DSharpPlus/DSharpPlus</RepositoryUrl>
-        <RepositoryType>Git</RepositoryType>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <PackageIcon>dsharpplus.png</PackageIcon>
-        <PackageIconUrl>https://raw.githubusercontent.com/DSharpPlus/DSharpPlus/master/logo/dsharpplus.png</PackageIconUrl>
-        <GenerateDocumentationFile>True</GenerateDocumentationFile>
-        <NoWarn>CS1591</NoWarn>
-        <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
-        <ManagePackageVersionsCentrally>True</ManagePackageVersionsCentrally>
-
-        <!-- SourceLink -->
-        <PublishRepositoryUrl>true</PublishRepositoryUrl>
-        <EmbedUntrackedSources>true</EmbedUntrackedSources>
-        <IncludeSymbols>true</IncludeSymbols>
-        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-        <Optimize>False</Optimize>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' != ''">
-        <Version>$(VersionPrefix)-$(VersionSuffix)-$(BuildNumber)</Version>
-        <AssemblyVersion>$(VersionPrefix).$(BuildNumber)</AssemblyVersion>
-        <FileVersion>$(VersionPrefix).$(BuildNumber)</FileVersion>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' == ''">
-        <Version>$(VersionPrefix)-$(VersionSuffix)</Version>
-        <AssemblyVersion>$(VersionPrefix).0</AssemblyVersion>
-        <FileVersion>$(VersionPrefix).0</FileVersion>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="'$(VersionSuffix)' == ''">
-        <Version>$(VersionPrefix)</Version>
-        <AssemblyVersion>$(VersionPrefix).0</AssemblyVersion>
-        <FileVersion>$(VersionPrefix).0</FileVersion>
-    </PropertyGroup>
-
-    <ItemGroup>
-        <None Include="../logo/dsharpplus.png" Pack="true" PackagePath="/"/>
-    </ItemGroup>
-
-  <ItemGroup>
-    <!-- We want to remove JSImportGenerator by default, becuase it takes a good deal of compilation time -->
-    <RemoveAnalyzer Include="Microsoft.Interop.JavaScript.JSImportGenerator" />
-  </ItemGroup>
-
+  <!-- Build -->
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <IsPackable>false</IsPackable>
+    <LangVersion>Latest</LangVersion>
+    <ManagePackageVersionsCentrally>True</ManagePackageVersionsCentrally>
+    <NoWarn>CS1591</NoWarn>
+    <Nullable>enable</Nullable>
+    <ProjectRoot>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), "*.sln"))</ProjectRoot>
+    <TargetFramework>net7.0</TargetFramework>
+    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
+  </PropertyGroup>
+  <!-- Nuget -->
```

Comment:
there should be a newline

Reply:
My formatter literally does not allow for this. I cannot save the file without the formatter removing the newlines.

Reply:
that is an issue unto its own, but does not excuse not doing it. if theres a separate reason then maybe

Reply:
`nano` it is then...